content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1696 @@
+
+
+<!-------------------------------------------------------------->
+
+## Quick start
+
+{{< tabs name="Python runtime setup" >}}
+{{% tab name="Windows" %}}
+
+For Windows, open a command prompt. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Run the `o3de.bat` batch file directly.
+    ```cmd
+    <engine>\scripts\o3de.bat <command>
+    ```
+
+- Launch python and run the `o3de.py` script.
+    ```cmd
+    <engine>\python\python.cmd <engine>\scripts\o3de.py <command>
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```cmd
+<engine>\scripts\o3de.bat register --this-engine
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+For Linux, open a terminal. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Run the `o3de.sh` shell script file directly.
+    ```bash
+    <engine>/scripts/o3de.sh <command>
+    ```
+
+- Launch python and run the `o3de.py` script.
+    ```bash
+    <engine>/python/python.sh <engine>/scripts/o3de.py <command>
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```bash
+<engine>/scripts/o3de.sh register --this-engine
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+<!-------------------------------------------------------------->
+
+## Commands
+The `o3de` Python script contains the following commands, with further details in the following sections. 
+
+| Command | Description | 
+| - | - |
+| [`get-global-project`](#get-global-project) | Gets the global project that is registered to the engine. |
+| [`set-global-project`](#set-global-project) | Sets the specified project as the engine's global project. |
+| [`create-template`](#create-template) | Creates a template out of the specified source path. |
+| [`create-from-template`](#create-from-template) | Creates an instance from a generic template.  |
+| [`create-project`](#create-project) | Creates a new project. |
+| [`create-gem`](#create-gem) | Creates a new Gem. |
+| [`register`](#register) | Registers an O3DE object. |
+| [`register-show`](#register-show) | Shows the registered O3DE objects. |
+| [`get-registered`](#get-registered) | Shows the path to the registered O3DE object with the specified name. |
+| [`enable-gem`](#enable-gem) | Enables the Gem in your project. |
+| [`disable-gem`](#disable-gem) | Disables the Gem in your project. |
+| [`edit-engine-properties`](#edit-engine-properties) | Edits the engine's properties. |
+| [`edit-project-properties`](#edit-project-properties) | Edits the project's properties. |
+| [`edit-gem-properties`](#edit-gem-properties) | Edits the Gem's properties. |
+| [`sha256`](#sha256) | Creates a hash value for an O3DE object using the SHA-256 secure hash algorithm. |
```

Comment:
```suggestion
| [`sha256`](#sha256) | Creates a hash value for an O3DE object using SHA-256 (Secure Hash Algorithm 256). |
```
Revised slightly to align to the format I'm finding in [Wikipedia](https://en.wikipedia.org/wiki/SHA-2) and AWS style.